src/test/java/core/basesyntax/strategy/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.dto.FruitDto;
+import core.basesyntax.model.Fruit;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static OperationsHandler operationsHandler;
+    private static FruitDto fruitDto;
+
+    @BeforeClass
+    public static void beforeClass() {
+        operationsHandler = new BalanceOperationHandler();
+        Storage.storage.put(new Fruit("banana"), 60);
+    }
```

Comment:
I think it's better to use setUp instead of beforeClass because we might have extra cases that need data from Storage.
same in other classes.